phpunit-plugin-bootstrap.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+require_once( __DIR__ . '/lib/antecedent/patchwork/Patchwork.php' );
```

Comment:
I agree, this would be useful for more things than just patchwork. Lets add that in.